content/en/getting_started/containers/datadog_operator.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Getting Started with the Datadog Operator
+kind: documentation
+further_reading:
+- link: "/agent/kubernetes/integrations/"
+  tag: "Documentation"
+  text: "Create and load an Autodiscovery Integration Template"
+- link: "/agent/guide/ad_identifiers/"
+  tag: "Documentation"
+  text: "Match a container with the corresponding Integration Template"
+- link: "/agent/guide/autodiscovery-management/"
+  tag: "Documentation"
+  text: "Manage which Container to include in the Agent Autodiscovery"
+- link: "/agent/kubernetes/tag/"
+  tag: "Documentation"
+  text: "Dynamically assign and collect tags from your application"
+- link: "/integrations/faq/integration-setup-ecs-fargate/?tab=rediswebui"
+  tag: "faq"
+  text: "Integration Setup for ECS Fargate"
+- link: "/agent/guide/secrets-management/"
+  tag: "Documentation"
+  text: "Secrets Management"
+---
+
+This guide describes the Datadog Operator, how to install it, and how to use it to install the Datadog Agent on Kubernetes.
+
+## What is the Datadog Operator? 
+
+The Datadog Operator is an open source [Kubernetes Operator][4] that enables you to deploy and configure the Datadog Agent in a Kubernetes environment. By using the Operator, you can use a single Custom Resource Definition (CRD) to deploy the node-based Agent, Cluster Agent, and Cluster Checks Runners. The Operator reports deployment status, health, and errors in the Operator's CRD status. Because the Operator uses higher-level configuration options, it limits the risk of misconfiguration.
```

Comment:
```suggestion
The Datadog Operator is an open source [Kubernetes Operator][4] that enables you to deploy and configure the Datadog Agent in a Kubernetes environment. By using the Operator, you can use a single Custom Resource Definition (CRD) to deploy the node-based Agent, Cluster Agent, and Cluster Checks Runner. The Operator reports deployment status, health, and errors in the Operator's CRD status. Because the Operator uses higher-level configuration options, it limits the risk of misconfiguration.
```
Cluster Checks Runner is singular here: https://docs.datadoghq.com/agent/cluster_agent/clusterchecksrunner/?tab=operator